ale/base/data_isis.py

```diff
@@ -413,7 +381,7 @@ def _sclock_hex_string(self):
             The hex string representation of the image
             start time as a double
         """
-        for key in self.naif_keywords:
+        for key in self.isis_naif_keywords:
             if re.match('CLOCK_ET_.*_COMPUTED', key[0]):
```

Comment:
You might want to compile the regex outside the loop and then use it to search inside. This is a very minor performance thing that is likely not needed though.